script/Arcueid/Day 5/05_00_ARC05_9_2B.txt

```diff
@@ -6,82 +6,85 @@ I stoop down in front of Arcueid.
 [sha:fa1e5311dfe889bb0a8186830ca0c5ba26f79877]{
 -- Page 10022, Offset 10422.
 -- 膝と両手を地面につく。
-I put my hands and knees on the earth.
+I put my hands and knees on the ground.
```

Comment:
I'd say revert, so much more dry.

Reply:
The sentence is sort of dry in general.
I think I should change it more.

>I stoop down in front of Arcueid.
>My hands and knees settle on the ground.

Just to give the sense of a continuous action.

Reply:
Maybe transpose it a bit?
> I feel the earth beneath my hands and knees.